_inc/lib/admin-pages/class.jetpack-my-jetpack-page.php

```diff
@@ -46,11 +46,31 @@ function jetpack_my_jetpack_change_user() {
 				Jetpack::init()->stat( 'admin', 'change-primary-successful' );
 				Jetpack::init()->do_stats( 'server_side' );
 
+				// Change the blog owner dotcom side
+				$this->wpcom_switch_blog_owner( $new_master_user );
 			}
 		}
 	}
 
 	/*
+	 * Tell wpcom that the master user has switched
+	 * so we can update the 'wpcom_blog_owner'
+	 */
+	function wpcom_switch_blog_owner( $new_master ) {
+		$request = array(
+			'new_blog_owner' => $new_master
+		);
+
+		// Tell wpcom about the change
+		Jetpack::load_xml_rpc_client();
+		$xml = new Jetpack_IXR_Client( array(
+			'user_id' => get_current_user_id(),
+		) );
+
+		$xml->query( 'jetpack.switchMasterUser', $request );
```

Comment:
A bit of a naming conflict -- the method refers to blog owner, endpoint master user.  Can we consolidate?